sklearn/neighbors/lof.py

```diff
@@ -144,6 +144,20 @@ class LocalOutlierFactor(NeighborsBase, KNeighborsMixin, UnsupervisedMixin,
         case, the offset is defined in such a way we obtain the expected
         number of outliers in training.
 
+    Examples
+    --------
+    >>> import numpy as np
```

Comment:
Maybe a simpler example:

```py
>>> import numpy as np
>>> from sklearn.ensemble import IsolationForest
>>> from sklearn.neighbors import LocalOutlierFactor
>>> X = [[-1.1], [0.2], [101.1], [0.3]]
>>> LocalOutlierFactor(n_neighbors=2).fit_predict(X)
array([ 1,  1, -1,  1])
```

Reply:
Surely that's shorter (and it would be even more without importing `IsolationForest`), but it looks much less intuitive to interpret the result, at least for me.

Reply:
We can include the `negative_outlier_factor_` as well:

```py
>>> import numpy as np
>>> from sklearn.neighbors import LocalOutlierFactor
>>> X = [[-1.1], [0.2], [101.1], [0.3]]
>>> clf = LocalOutlierFactor(n_neighbors=2)
>>> clf.fit_predict(X)
array...
>>> clf.negative_outlier_factor_
array...
```

Reply:
Ok, I've taken this code now, with ellipses added to prevent precision errors on different platforms.